azurerm/provider.go

```diff
@@ -217,6 +217,7 @@ func Provider() terraform.ResourceProvider {
 		"azurerm_batch_certificate":                                  resourceArmBatchCertificate(),
 		"azurerm_bot_channel_email":                                  resourceArmBotChannelEmail(),
 		"azurerm_bot_channel_slack":                                  resourceArmBotChannelSlack(),
+		"azurerm_bot_channel_msteams":                                resourceArmBotChannelMsTeams(),
```

Comment:
It looks like the UI has this as MSTeams but would it be more readable to split it out a little here to look like `azurerm_bot_channel_ms_teams`?

Reply:
it might be worth making this `microsoft_teams` rather than `ms` so this is clearer - WDYT?

Reply:
If we look at sources like this
1 - https://github.com/Azure/azure-sdk-for-go/blob/7a9d2769e4a581b0b1bc609c71b59af043e05c98/services/preview/botservice/mgmt/2018-07-12/botservice/models.go
and 2 - https://docs.microsoft.com/en-us/cli/azure/bot/msteams?view=azure-cli-latest

It seems like "MsTeams" is the consistent use Microsoft themselves make - might be better if we keep it this way, especially if someone is migrating from something else?

Reply:
I agree that it should be kept as MsTeams but will push again that it should be `azurerm_bot_channel_ms_teams`. Microsoft makes the distinction between MsTeams with capitals more often than not and we can mirror that but splitting out the words from each other